### PR TITLE
OCPBUG#10295:Modified the note related to network policy.

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.10+

Issue:
[OCPBUGS-10295](https://issues.redhat.com/browse/OCPBUGS-10295)

Link to docs preview:
[Doc Preview](https://60113--docspreview.netlify.app/openshift-enterprise/latest/networking/network_policy/about-network-policy.html#nw-networkpolicy-about_about-network-policy)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/nw-networkpolicy-about.adoc [hunk 1]
```diff
@@ -12,7 +12,9 @@ In {product-title} {product-version}, OpenShift SDN supports using network polic
 
 [WARNING]
 ====
-Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules.
+Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules. However, if they are connected to pods with host network policies, they might be affected by the network policy rules.
```
Comment (asood-rh) [inline modules/nw-networkpolicy-about.adoc:15]: However, if they are connected to pods with host network policies, they might be affected by the network policy rules.

Rephrase it to. 
However pods connecting to the  host networked pods might be affected by the network policy rules.

File: modules/nw-networkpolicy-about.adoc [hunk 2]
```diff
@@ -12,7 +12,9 @@ In {product-title} {product-version}, OpenShift SDN supports using network polic
...
+
+Pods currently cannot block traffic from localhost, nor from their resident nodes.
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/nw-networkpolicy-about.adoc:17]: Can you check that once from the K8s doc. Here they had mentioned - "The ability to prevent loopback or incoming host traffic (Pods cannot currently block localhost access, nor do they have the ability to block access from their resident node)."
https://kubernetes.io/docs/concepts/services-networking/network-policies/

File: modules/nw-networkpolicy-about.adoc (hunk 1 above)
Comment (subhtk) [reply modules/nw-networkpolicy-about.adoc:15]: Ack.

File: modules/nw-networkpolicy-about.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (asood-rh) [reply modules/nw-networkpolicy-about.adoc:17]: @subhtk It is OK if you want to use the sentence if the K8s doc is using it. But if you look at the bug itself the idea is to clarify the functionality of network policies. The traffic blocking is done either  network policies or the iptable rules on the node itself.

[the remark this answers is not in the document]
Comment (asood-rh) [reply modules/nw-networkpolicy-about.adoc:17]: Qualification could be - Loopback or incoming host traffic blocking might be done by implementing workarounds using Operating System components (such as SELinux, OpenVSwitch, IPTables, and so on) or Layer 7 technologies (Ingress controllers, Service Mesh implementations) or admission controllers.

[the remark this answers is not in the document]
Comment (subhtk) [reply modules/nw-networkpolicy-about.adoc:17]: @asood-rh Acknowledged. I will change it to Network policies.

**Review by bergerhoffer — COMMENTED** [review]
Two comments!

File: modules/nw-networkpolicy-about.adoc [hunk 3]
```diff
@@ -12,7 +12,9 @@ In {product-title} {product-version}, OpenShift SDN supports using network polic
 
 [WARNING]
 ====
-Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules.
+Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules. However pods connecting to the host networked pods might be affected by the network policy rules.
+
+Network Policies currently cannot block traffic from localhost, nor from their resident nodes.
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/nw-networkpolicy-about.adoc:17]: Ack

**Comment by ousleyp** [comment]
(LGTM as per our Slack chat :))
